File: a8.cpp
// program to compare three numbers and display maximum
#include<iostream>
using namespace std;
int main()
{
    int a, b, c;
    cout<<"Enter First no. : ";
    cin>>a;
     cout<<"Enter Second no. : ";
    cin>>b;
     cout<<"Enter Third no. : ";
    cin>>c;
    if (a>b)
    {
        if (a>c)
        {
            cout<<a<<" is maximum"<<endl; 
        }
        else
        {
            cout<<c<<" is maximum"<<endl; 
        }
    }
    else
        {
            if (b>c)
        {
            cout<<b<<" is maximum"<<endl; 
        }
        else
        {
            cout<<c<<" is maximum"<<endl; 
        }
        }
        return 0;
}   

File: a8.java
// program to compare three numbers and display maximum
import java.util.Scanner;
class a8{
    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);
        int a, b, c;
        System.out.println("Enter First no. : ");
        a = sc.nextInt();
        System.out.println("Enter Second no. : ");
        b = sc.nextInt();
        System.out.println("Enter Third no. : ");
        c = sc.nextInt();
        
        if(a>b)
        {
            if(a>c)
            {
                System.out.println(a+" is maximum");
            }
            else
            {
                System.out.println(c+" is maximum");
            }
        }
        else
        {
                if(b>c)
                {
                    System.out.println(b+" is maximum");
                }
                else
                {
                    System.out.println(c+" is maximum");
                }
        }
       
    }
}